src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java

```diff
@@ -45,6 +47,17 @@ public static void logMessageAndHalt(Request request, int statusCode, String mes
         }
     }
 
+    /** Utility method to parse generic object from Http response. */
+    public static <T> T getPOJOFromHttpResponse(HttpResponse<String> response, Class<T> clazz) {
+        T pojo = null;
+        try {
+            pojo = mapper.readValue(response.body(), clazz);
+        } catch (JsonProcessingException e) {
+            LOG.error("Unable to get POJO from http response", e);
```

Comment:
Can you add the clazz name to this error output to help with debugging?

Reply:
Updated.